telemetry/definitions/commonDefinitions.json

```diff
@@ -3505,6 +3524,10 @@
                     "type": "codeTransformLocalMavenVersion",
                     "required": false
                 },
+                {
+                    "type": "codeTransformPlanReceivedCount",
```

Comment:
You have to add all distinct types into the types section at the top of the file.

Reply:
This has been addressed in a more recent commit.